src/styles/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
```

Comment:
add `padding` to `.card`, remove `height` and set `width: 100%` here

Reply:
> add `padding` to `.card`, remove `height` and set `width: 100%` here

If I remove "height", my tests fail.